server/integration/infinispan/src/main/resources/org/jboss/as/clustering/infinispan/subsystem/LocalDescriptions.properties

```diff
@@ -545,6 +543,7 @@ datagrid-infinispan.leveldb-store.remove=Removes a cache leveldb store configura
 datagrid-infinispan.leveldb-store.block-size=The cache store block size
 datagrid-infinispan.leveldb-store.cache-size=Cache size for the cache store
 datagrid-infinispan.leveldb-store.clear-threshold=Cache store cache clear threshold
+datagrid-infinispan.expiration.queue-size=Leveldb store expiration queue size
```

Comment:
@tristantarrant The LevelDB expiration queue-size attribute was "reusing" the `datagrid-infinispan.clustered-cache.queue-size` description, and `datagrid-infinispan.leveldb-store.expiration.queue-size` didn't work. Could you have a look and maybe create a separate JIRA?